core/src/main/java/org/apache/accumulo/core/fate/AbstractFateStore.java

```diff
@@ -52,7 +52,7 @@ public abstract class AbstractFateStore<T> implements FateStore<T> {
   private static final Logger log = LoggerFactory.getLogger(AbstractFateStore.class);
 
   protected final Set<Long> reserved;
-  protected final Map<Long,Long> defered;
+  protected final Map<Long,Long> deferred;
```

Comment:
This is refactored from the ZooStore and is actually misspelled in 2.1 and main as well so we should fix it there too. https://github.com/apache/accumulo/blob/d97317945290fcd4e3a4eb8b3cf7e427fcc322bd/core/src/main/java/org/apache/accumulo/core/fate/ZooStore.java#L64

Reply:
Looks like the same thing was fixed in #4126 which is waiting to be merged into 2.1 so I think we can just let it be handled there if it ends up being merged.

Reply:
Added #4128 to fix the things mentioned in 2.1